wynnset/tapestry-wp
# 435 Planet View Updates

Closes #435 
Adds:
 - Check-mark icon to completed modules in planet view
Changes:
 - Hides planet view progress bar if progress is 0
 - Hides planet view icon if no image URL exists

Steps to test:
 - Create a TYDE tapestry with a module, stage and 2 topics
 - Leave both planet view icon links empty for the module
 - Verify the progress bar is not visible before completing any topics
 - Complete one topic and verify the progress bar has appeared, but without the icon
 - Complete the other topic and verify there is a check-mark in the bottom corner of the planet
 - Add a Planet View Completed icon link to the module and verify the icon appears
 - Add a new topic to the stage
 - Add a Planet View Incomplete icon link to the module and verify the icon appears
